src/ape/managers/networks.py

```diff
@@ -20,6 +18,9 @@ class NetworkManager:
     active_provider: Optional[ProviderAPI] = None
     _default: Optional[str] = None
 
+    class Config:
+        keep_untouched = (cached_property,)  # https://github.com/samuelcolvin/pydantic/issues/1241
+
     @cached_property
     def ecosystems(self) -> Dict[str, EcosystemAPI]:
```

Comment:
One thing I noticed is that we are passing `self.config.REQUEST_HEADER` here (which is of type `Dict`) but the `EcosystemAPI` class expected a `str` for `request_header`.. What is going on there?

Reply:
honestly... I forget :see_no_evil: